### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Kaijou88**

### CODE REVIEW

File: src/main/java/spring/intro/config/WebConfig.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.intro.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```
Comment (boroda4436) [inline src/main/java/spring/intro/config/WebConfig.java:16]:
```suggestion
    public InternalResourceViewResolver getResolver() {
```

File: src/main/java/spring/intro/controllers/UserController.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                .findFirst()
+                .get();
+    }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAll() {
+        List<User> users;
+        users = userService.listUsers();
+        for (User user : users) {
+            UserResponseDto userDto = new UserResponseDto();
```
Comment (boroda4436) [inline src/main/java/spring/intro/controllers/UserController.java:44]: move to a separate method

File: src/main/java/spring/intro/controllers/UserController.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                .filter(u -> u.getId().equals(userId))
+                .findFirst()
+                .get();
+    }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAll() {
+        List<User> users;
+        users = userService.listUsers();
+        for (User user : users) {
```
Comment (boroda4436) [inline src/main/java/spring/intro/controllers/UserController.java:43]: use Stream API

File: src/main/java/spring/intro/controllers/UserController.java (hunk 2 above)
Comment (Kaijou88) [reply src/main/java/spring/intro/controllers/UserController.java]: done

File: src/main/java/spring/intro/controllers/UserController.java (hunk 3 above)
Comment (Kaijou88) [reply src/main/java/spring/intro/controllers/UserController.java]: done

**Pull request merged**